prac_06/guitar_test.py

```diff
@@ -0,0 +1,19 @@
+from prac_06.guitar import Guitar
+
+
+def run_tests():
+    year = 1922
+    cost = 16035.40
+    name = "Gibson L-5 CES"
```

Comment:
Not a big issue, but perhaps labelling these in the same order as inputted into the class?